src/rust/bitbox02-rust/src/workflow/trinary_choice.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 Shift Crypto AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::bb02_async::option;
+use core::cell::RefCell;
+
+extern crate alloc;
+use alloc::boxed::Box;
+
+use bitbox02::ui::trinary_choice_create;
+pub use bitbox02::ui::TrinaryChoice;
+
+pub async fn trinary_choice(
```

Comment:
I'd name it `choose`. Would read nicer and more consistent: verb vs noun, like `trinary_input_string::enter(...)` or `menu::pick(...)`.

Reply:
Done